tests/x2c-perturb_h/input.dat

```diff
@@ -0,0 +1,28 @@
+#! Test SFX2C-1e with a static electric field on He aug-cc-pVTZ
+
+eref_x2c_0 = -2.8613053511779594
+eref_x2c_p = -2.8679759489083452
+
+molecule {
+  0 1
+  He 0.0 0.0 0.0
+  symmetry c1  # field breaks symmetry
```

Comment:
should we arrange for the presence of a field to impose c1/no_com/no_reorient on the mol? does the field ever not break mol symmetry and we want to preserve symm?

Reply:
I think there are cases when you might want the user to determine how to lower the symmetry. And in some cases fields do not break symmetry (e.g. filed in the z direction applied to water, C2v). There is some code to make sure that the computation is setup correctly, e.g. (around line 1321 of `mintshelper.cc`)
```
    if (lambda[0] != 0.0) {
        if (msymm.component_symmetry(0) != 0) {
            outfile->Printf("  WARNING: Requested mu(x) perturbation, but mu(x) is not symmetric.\n");
        } else {
            outfile->Printf("  Perturbing V by %f mu(x).\n", lambda[0]);
            dipoles[0]->scale(lambda[0]);
            potential_mat->add(dipoles[0]);
        }
    }
```